parsers/IL.py

```diff
@@ -26,21 +26,8 @@
 TZ = "Asia/Jerusalem"
 
 
-def flatten_list(_2d_list) -> list:
-    flat_list = []
-    for element in _2d_list:
-        if type(element) is list:
-            for item in element:
-                flat_list.append(item)
-        else:
-            flat_list.append(element)
-    return flat_list
-
-
-def fetch_production(zone_key="IL", session=None, target_datetime=None, logger=None) -> dict:
-    if target_datetime:
-        raise NotImplementedError('This parser is not yet able to parse past dates')
-        
+def fetch_all() -> list:
```

Comment:
Suggestion for improvement (although it's a bit out of scope):
Would you mind changing the selectors from finding all `.statusVal`s to use the more stable selectors (`#lblActualSupply`,`#lblCurrentSupply`)? That way we're not relying on the order (and have to remove an entry in the middle) and the function could then return a dictionary with `{production: X, consumption: Y}` instead of a list :)

Reply:
Maybe I can do it, but as sayed it is out of the scope of this PR.
will open a seperate one for it soon.. 👍